staging/src/k8s.io/kubectl/pkg/cmd/apply/apply.go

```diff
@@ -1062,6 +1105,7 @@ const (
 	fieldManagerServerSideApply = "kubectl"
```

Comment:
I opted not to add a custom type for these because `FieldManager` is used in a lot of places and I think that kind of refactor is outside of the scope of this PR, and having it typed only here meant converting to a string for the helper methods which felt messy.